src/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -0,0 +1,136 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#include "jitpch.h"
+#ifdef _MSC_VER
+#pragma hdrstop
+#endif
+
+#ifdef FEATURE_HW_INTRINSICS
+
+#include "emit.h"
+#include "codegen.h"
+#include "sideeffects.h"
+#include "lower.h"
+#include "gcinfo.h"
+#include "gcinfoencoder.h"
+
+//------------------------------------------------------------------------
+// genIsTableDrivenHWIntrinsic:
+//
+// Arguments:
+//    category - category of a HW intrinsic
+//
+// Return Value:
+//    returns true if this category can be table-driven in CodeGen
+//
+static bool genIsTableDrivenHWIntrinsic(NamedIntrinsic intrinsicId, HWIntrinsicCategory category)
+{
+    // TODO - make more categories to the table-driven framework
```

Comment:
This should be more specific - in this case probably `TODO-Arm64-Cleanup` (https://github.com/dotnet/coreclr/blob/master/Documentation/coding-guidelines/clr-jit-coding-conventions.md#7.1.5)

Reply:
Fixed locally.